integrations/_infra/Dockerfile

```diff
@@ -1,28 +1,58 @@
-FROM python:3.11-slim-bookworm
+FROM python:3.11-alpine AS base
 
 ARG BUILD_CONTEXT
+
+ENV LIBRDKAFKA_VERSION=1.9.2
+
+# Install system dependencies and libraries
+RUN apk add --no-cache \
+    gcc \
+    musl-dev \
+    build-base \
+    bash \
+    oniguruma-dev \
+    make \
+    autoconf \
+    automake \
+    libtool \
+    curl \
+    libffi-dev && \
+        echo "@edge http://dl-cdn.alpinelinux.org/alpine/edge/main" >> /etc/apk/repositories \
+        && echo "@edgecommunity http://dl-cdn.alpinelinux.org/alpine/edge/community" >> /etc/apk/repositories \
+        && apk add --no-cache alpine-sdk "librdkafka@edgecommunity>=${LIBRDKAFKA_VERSION}" "librdkafka-dev@edgecommunity>=${LIBRDKAFKA_VERSION}" && curl -sSL https://install.python-poetry.org | python3 - \
+        && /root/.local/bin/poetry config virtualenvs.in-project true
+
+WORKDIR /app
+
+COPY ./${BUILD_CONTEXT}/pyproject.toml ./${BUILD_CONTEXT}/poetry.lock /app/
+
+RUN /root/.local/bin/poetry install --without dev --no-root --no-interaction --no-ansi --no-cache && pip cache purge
+
+FROM python:3.11-alpine AS prod
+
 ARG INTEGRATION_VERSION
+ARG BUILD_CONTEXT
 
 LABEL INTEGRATION_VERSION=${INTEGRATION_VERSION}
 # Used to ensure that new integrations will be public, see https://docs.github.com/en/packages/learn-github-packages/configuring-a-packages-access-control-and-visibility
-LABEL org.opencontainers.image.source https://github.com/port-labs/ocean
+LABEL org.opencontainers.image.source=https://github.com/port-labs/ocean
 
-ENV LIBRDKAFKA_VERSION 1.9.2
+# Install only runtime dependencies
+RUN apk add --no-cache \
+    librdkafka-dev \
+    bash \
+    oniguruma-dev
 
 WORKDIR /app
 
-RUN apt update && \
-    apt install -y wget make g++ libssl-dev autoconf automake libtool curl librdkafka-dev && \
-    apt-get clean
-
-COPY ./integrations/_infra/init.sh /app/init.sh
-
-RUN chmod +x /app/init.sh
+# Copy dependencies from the build stage
+COPY --from=base /app /app
 
+# Copy the application code
 COPY ./${BUILD_CONTEXT} /app
 
-COPY ./integrations/_infra/Makefile /app/Makefile
-
-RUN export POETRY_VIRTUALENVS_CREATE=false && make install/prod && pip cache purge
+# Clean up old setuptools + update certificates if needed
+RUN pip uninstall -y setuptools || true && test -e /usr/local/share/ca-certificates/cert.crt && update-ca-certificates || true
 
-ENTRYPOINT ./init.sh
+# Run the application
+CMD ["/app/.venv/bin/python", "/app/.venv/bin/ocean", "sail"]
```

Comment:
maybe worth adding some `alias` for running `ocean` wdyt? so if someone is executing into the container, it will be easy to use the installed `ocean`

Reply:
Sure thang